tutorials/io/ntuple/ntpl002_vector.C

```diff
@@ -28,7 +28,7 @@
 #include <utility>
 
 // Import classes from experimental namespace for the time being
-using RNTupleModel = ROOT::Experimental::RNTupleModel;
+using RNTupleModel = ROOT::RNTupleModel;
```

Comment:
The remark to qualify the classes applies to all tutorials, not just the first one...